tests/README.md

```diff
@@ -0,0 +1,90 @@
+MetalK8s is an infrastructure product, that rely on ansible so framework in python is somewhat “obvious”.
+Pytest is quite famous right famous.
+
+Test to be run could be splitted in different category:
+
+1. Test that run before installation on any machine
+    - test of os-check (centos check and kernel)
+    - test first installation of component that doesn't need kubernetes (node_exporter)
+    - test of the complete installation it-self
+
+2. Test that run after the installation of kubernetes only (not the entire set of MetalK8s):
+    - first installation of elasticsearch
+
+3. Test that could be run at each point of time after MetalK8s installation (non-destructive):
+    - test of prometheus monitoring
+
+4. Test that meant to be run after installation and before production (conformance test)
+    - test kubernetes object creation
+
+5. Test that need to run on non-production environnement (CI test)
+    - test of failover
+
+6. Test that could run with only mocked environement:
+    - inventory check
+
+7. Unittesting of ansible roles: molecule
+
+```
+pytest --inventory <path_to_inventory> --install --conformance --go-destructive
+```
+
+Command line arguments:
+
+* `--install` trigger installation test and fixture (handles `.1` and `.2` test category)
+
+* `--conformance` enable conformance test (handles `.4`)
+
+* `--go-destructive` enable full testing, including test of failover (handles `.5`)
+
+No args on pytest mean only non-destructive test (handle case `.3` and `.6`)
+
+This represents e2e/integration testing of MetalK8s. Might be better to
+have unittesting as a separate test-suite (Molecule, different python scripts in the repo)
```

Comment:
Yeah, I looked at how to 'integrate' Molecule tests in pytest, but didn't find a readily-available solution. Nonetheless, I'd like to see something like that happening, could be a nice little pytest plugin.